src/stache/includes/cli/checkout.html

```diff
@@ -1 +1 @@
-<sky-code>--checkout</sky-code> — Specifies the branch to clone from the template. The default is <sky-code>master</sky-code>.
+<sky-code>--checkout</sky-code> — Specifies the branch to clone from the template. The default is the versioned branch with the <sky-code>[MAJOR_VERSION].x.x</sky-code> format that is required in the template's repo.
```

Comment:
This sentence was a bit hard to read.  We also need to mention it's the `MAJOR_VERSION` associated the SKY UX CLI installed.  Perhaps something like:

The default branch is controlled by the version of the SKY UX CLI installed using the following format: `[SKYUX CLI Major Version].x.x`.

Reply:
Updated this and the corresponding note at the bottom of the page.